finagle-thrift/src/main/scala/com/twitter/finagle/thrift/ThriftClientBufferedCodec.scala

```diff
@@ -26,7 +26,15 @@ object ThriftClientBufferedCodec {
   /**
    * Create a [[com.twitter.finagle.thrift.ThriftClientBufferedCodecFactory]]
    */
-  def apply() = new ThriftClientBufferedCodecFactory
+  def apply(): ThriftClientBufferedCodecFactory =
+    apply(Protocols.binaryFactory(), true)
+
+  def apply(protocolFactory: TProtocolFactory): ThriftClientBufferedCodecFactory =
+    apply(protocolFactory, true)
+
+  def apply(protocolFactory: TProtocolFactory,
```

Comment:
Okay. I chatted with a people at the team. Let's drop the codec support for now and only add stack param available via the `with`-method.